tools/buildsys/src/main.rs

```diff
@@ -167,7 +167,9 @@ fn build_variant() -> Result<()> {
 
     if let Some(packages) = manifest.included_packages() {
         let image_format = manifest.image_format();
-        VariantBuilder::build(&packages, image_format).context(error::BuildAttempt)?;
+        let supported_arches = manifest.supported_arches();
+        VariantBuilder::build(&packages, image_format, supported_arches)
+            .context(error::BuildAttempt)?;
```

Comment:
We also want the supported arches check when building packages.

Reply:
But we don't have that information when building packages, do we, since it comes from the metadata in the variant's cargo.toml?

Reply:
We are able to get the information we need from the environment! :)